include/range/v3/view/iota.hpp

```diff
@@ -523,6 +523,34 @@ namespace ranges
         /// \ingroup group-views
         RANGES_INLINE_VARIABLE(closed_iota_fn, closed_iota)
 
+        /// # ranges::views::ints
+        /// The ints view returns a range of monotonically increasing ints.
+        ///
+        /// ## Example
+        /// \snippet example/view/ints.cpp ints example
+        ///
+        /// ### Output
+        /// \include example/view/ints_golden.txt
+        ///
+        /// ## Syntax
+        /// ```cpp
+        /// auto output_range = ranges::views::ints(lower_bound, upper_bound);
+        /// ```
+        ///
+        /// ## Parameters
+        /// <pre><b>lower_bound</b></pre>
+        ///   - Optional lower bound
+        ///
+        /// <pre><b>upper_bound</b></pre>
+        ///   - Exclusive upper bound
+        ///   - Required when `lower_bound` is specified
+        ///     - To create an infinite range with a `lower_bound`, use
+        ///     `ranges::unreachable` as the `upper_bound`
+        ///
+        /// <pre><b>output_range</b></pre>
+        ///   - Range of monotonically increasing ints
+        ///   - When an `upper_bound` is not specified, the range is quasi-infinite
```

Comment:
Is "quasi-infinite" defined anywhere? What does it mean? What concepts does the resulting range satisfy?

Reply:
I based this off of the existing docs "Generate a range of monotonically increasing ints. When used without arguments, it generates the quasi-infinite range". If I'm reading the code correctly, it looks like the iterator will eventually overflow.

Not sure what concepts it satisfies or how to find out. Can you point me to that info?

Reply:
All "infinite" ranges have difference types that will eventually overflow. It's a problem with the ranges design I was never able to properly solve.